src/server/rdb_load.cc

```diff
@@ -1331,15 +1331,15 @@ error_code RdbLoaderBase::ReadObj(int rdbtype, OpaqueObj* dest) {
       iores = ReadStreams(rdbtype);
       break;
     case RDB_TYPE_JSON:
-      iores = ReadJson();
```

Comment:
why did you change here and at line 1342?

Reply:
I removed the ReadJson as it is the same as ReadGeneric